libs/httpproxy/proxy-camel-route/src/main/java/org/apache/commons/httpclient/contrib/ssl/AuthSSLX509TrustManager.java

```diff
@@ -69,18 +69,20 @@ public AuthSSLX509TrustManager(final X509TrustManager defaultTrustManager) {
   /** @see javax.net.ssl.X509TrustManager#checkClientTrusted(X509Certificate[], String authType) */
   public void checkClientTrusted(X509Certificate[] certificates, String authType)
```

Comment:
✏️  If certificates can be null then it should be tagged with `@Nullable`

Reply:
@paouelle what should be imported for `@Nullable`?

Reply:
`import javax.annotation.Nullable;`